.cirrus.yml

```diff
@@ -14,12 +14,24 @@ unittest_task:
     - apt-get -y install llvm clang libelf-dev libpcap-dev python3-pip
     - rustup component add rustfmt
     - rustup component add clippy
+  before_cache_script: rm -rf $CARGO_HOME/registry/index
+
+unittest_task:
+  << : *cargo_task_template
   build_script: cargo build --verbose
   test_script: cargo test --verbose
   check_script:
     - cargo fmt --check
     - cargo clippy -- -D warnings
-  before_cache_script: rm -rf $CARGO_HOME/registry/index
+
+benchmark_task:
```

Comment:
thanks for the series, and thanks for updating here using templates.
I'll approve so this will be unblocked leaving some room for last-minute updates or comments.